yarn.lock

```diff
@@ -4656,7 +4656,7 @@ debug@=3.1.0:
   dependencies:
     ms "2.0.0"
 
-debug@^3.0.0, debug@^3.0.1, debug@^3.2.5, debug@^3.2.6:
+debug@^3.0.0, debug@^3.0.1, debug@^3.2.5:
```

Comment:
We have a dedicated task to update dependencies.

Reply:
yarn.lock is not up to date on master :/